docs/ops/activation/LogSoftmax_5.md

```diff
@@ -0,0 +1,42 @@
+## LogSoftMax <a name="LogSoftmax"></a> {#openvino_docs_ops_activation_LogSoftmax_5}
+
+**Versioned name**: *LogSoftmax-5*
+
+**Category**: *Activation*
+
+**Short description**: LogSoftmax computes the log of softmax values for the given input.
+
+**Attributes**
+
+* *axis*
+
+  * **Description**: *axis* represents the axis of which the *LogSoftmax* is calculated. *axis* equal 1 is a default value.
+  * **Range of values**: positive integer value
+  * **Type**: int
+  * **Default value**: 1
```

Comment:
Why is the default value equal to 1? I know that our SoftMax has the same it but in TF and ONNX specs they have -1. Please investigate what is better and correct value.

Reply:
Yes, I did like in our Softmax. Value 1 was chosen since axis is restricted to positive ints and since input is usually 2D. After some thinking, I decided to not restrict axis to positive in new op, better to support negative values as well.